package-lock.json

```diff
@@ -9,10 +9,12 @@
       "version": "0.0.0",
       "dependencies": {
         "@angular/animations": "^16.0.0",
+        "@angular/cdk": "^16.2.14",
         "@angular/common": "^16.0.0",
         "@angular/compiler": "^16.0.0",
         "@angular/core": "^16.0.0",
         "@angular/forms": "^16.0.0",
+        "@angular/material": "^16.2.14",
```

Comment:
read about this `^` 
 
 https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json